upload-api/functions/ucan-invocation-router.js

```diff
@@ -118,6 +118,17 @@ export const knownWebDIDs = {
  * @param {import('aws-lambda').APIGatewayProxyEventV2} request
  */
 export async function ucanInvocationRouter(request) {
+  // Capture X-Client custom header for analytics - ignore if not present
+  const clientId = request.headers['X-Client']
```

Comment:
Just incase `request.headers` is lower-cased or the client sends with weird casing e.g. `X-client`.

```suggestion
  const clientId = new URLSearchParams(request.headers).get('X-Client')
```

Reply:
This doesn't compile because this request.headers is type `APIGatewayProxyEventHeaders` which can't be passed to `URLSearchParams`